.travis.yml

```diff
@@ -7,7 +7,7 @@ notifications:
     on_failure: always
 env:
   global:
-    - ROS_REPO=ros
+    - ROS_REPO=ros-shadow-fixed
```

Comment:
It's now called `ros-testing` after the GPG key rotation

Reply:
Hm, the CI doc at https://github.com/ros-planning/moveit_ci mentions `ros` and `ros-shadow-fixed` as possible values.
Where does `ros-shadow-fixed` points towards now? Do http://packages.ros.org/ros-testing/ and http://packages.ros.org/ros-shadow-fixed/ contain the very same packages?

Reply:
For backward compatibility, it is both. But it was renamed when the GPG keys were rotated. Also cf.
https://github.com/ros-industrial/industrial_ci/blob/82154e0457a19c9f824560c57729091abdd436b6/industrial_ci/src/env.sh#L130-L132

Reply:
Alright, the CI indeed works with `ros-testing`.

From this check in the MoveIt CI: https://github.com/ros-planning/moveit_ci/blob/a18a9aff46800662db01a44bf31532b63c19c434/travis.sh#L42-L45 I assumed that an error will be thrown if values other than `ros` or `ros-shadow-fixed` are used. I opened https://github.com/ros-planning/moveit_ci/issues/87 to track the update to the CI and doc.

Reply:
NB: This repository uses industrial_ci, not moveit_ci - cf. https://github.com/ipa320/schunk_modular_robotics/blob/3ceb233cdef59d9611eb22203da4bc910bf60fd2/.travis.yml#L23

Reply:
NB: This repository uses industrial_ci, not moveit_ci - cf. https://github.com/ipa320/schunk_modular_robotics/blob/3ceb233cdef59d9611eb22203da4bc910bf60fd2/.travis.yml#L23

Reply:
Ups, you are right. That explains a lot :-) I got confused with another repository.